src/prefect/deployments/runner.py

```diff
@@ -467,6 +497,7 @@ def from_flow(
         work_queue_name: Optional[str] = None,
         job_variables: Optional[dict[str, Any]] = None,
         entrypoint_type: EntrypointType = EntrypointType.FILE_PATH,
+        sla: Optional[Union[SlaTypes, list[SlaTypes]]] = None,  # experimental
```

Comment:
Can you make sure that `sla` is included in the docstring for functions and methods where it's added and that the docstring mentions that it's experimental?

Reply:
Absolutely